pub/default/.htaccess

```diff
@@ -0,0 +1,234 @@
+############################################
```

Comment:
I'd prefer to symlink the .htaccess if possibile, it allows for less duplication on the repo